lib/werewolf/game.rb

```diff
@@ -533,7 +543,7 @@ def print_claims
 
     def print_tally
       changed
-      notify_observers(:action => 'tally', :vote_tally => vote_tally)
+      notify_observers(:action => 'tally', :vote_tally => vote_tally, :remaining_votes => remaining_votes)
```

Comment:
Like the output of `w tally`. I'd want to avoid the 'Show More' text if possible.